OpenSim/Common/AbstractProperty.h

```diff
@@ -104,6 +104,14 @@ class OSIMCOMMON_API AbstractProperty {
     // TODO: replace this with something more reasonable
```

Comment:
@carmichaelong do you think it makes sense to now remove this TODO?

What do you think about adding the `precision` option to `toString()`, with a default argument of 6 or 7? One issue there is that the deprecated properties would ignore the precision, but that is already the case with this implementation.

Reply:
I originally had it like that, but then all of the signatures of the deprecated properties would have to change. I figured this was better than doing that, but I'm happy to do that.

Reply:
Let's see what @aseth1 thinks.